app/src/main/res/raw-mcc510/abtesting.json

```diff
@@ -7,7 +7,7 @@
       {
         "experiment_name": "smart_shopping_copy_a",
         "bucket_range_start": 1,
-        "bucket_range_end": 7
+        "bucket_range_end": 4
```

Comment:
Do you accidentally change the value of smart shopping search?

Reply:
yes XD